beacon-chain/powchain/service.go

```diff
@@ -200,8 +190,16 @@ func NewService(ctx context.Context, config *Web3ServiceConfig) (*Service, error
 
 // Start a web3 service's main event loop.
 func (s *Service) Start() {
+	for {
```

Comment:
Can this timeout?

Reply:
I used a ticker instead, we would retry every 6 seconds. If we timeout, then the beacon node would not be able to process any deposits, search for eth1 blocks, etc in the future, if it doesn't 
connect with an eth1 node in 6 seconds